apps/mobile/src/components/Community/CommunityMeta.tsx

```diff
@@ -0,0 +1,225 @@
+            font={{ family: 'ABCDiatype', weight: 'Regular' }}
+          >
+            {community.creator.username}
+          </Typography>
+        </GalleryTouchableOpacity>
+      );
+    } else if (community.contractAddress) {
+      return (
+        <View className="flex flex-row space-x-1 items-center">
+          <RawProfilePicture size="xs" default eventElementId={null} eventName={null} />
+          <LinkableAddress
+            chainAddressRef={community.contractAddress}
+            type="Community Contract Address"
+            textStyle={{ color: colors.shadow }}
+          />
+        </View>
+      );
+    } else {
+      return null;
+    }
+  }, [community.creator, community.contractAddress, handleUsernamePress]);
+
+  return (
+    <View className="flex flex-row space-x-6">
+      {community?.chain !== 'POAP' && (
+        <View className="flex flex-column space-y-1">
+          <Typography
+            font={{ family: 'ABCDiatype', weight: 'Regular' }}
+            className="text-xs uppercase"
+          >
+            created by
+          </Typography>
+          {showAddressOrGalleryUser}
+        </View>
+      )}
+      <View className="flex flex-column space-y-1">
+        <Typography
+          font={{ family: 'ABCDiatype', weight: 'Regular' }}
+          className="text-xs uppercase"
+        >
+          network
+        </Typography>
+
+        {community.chain && (
+          <View className="flex flex-row space-x-1 items-center">
+            <NetworkIcon chain={community.chain} />
+            <Typography
+              font={{ family: 'ABCDiatype', weight: 'Regular' }}
+              className="text-sm text-shadow"
+            >
+              {community.chain}
+            </Typography>
+          </View>
+        )}
+      </View>
+      {ENABLED_TOTAL_TOKENS && (
+        <View className="space-y-1">
+          <Typography
+            font={{ family: 'ABCDiatype', weight: 'Regular' }}
+            className="text-xs uppercase text-right"
+          >
+            items
+          </Typography>
+
+          <Typography
+            font={{ family: 'ABCDiatype', weight: 'Regular' }}
+            className="text-sm text-shadow text-right"
+          >
+            {formattedTotalTokens}
+          </Typography>
+        </View>
+      )}
+      {totalPosts > 0 && isKoalaEnabled && (
+        <View className="flex flex-column space-y-1">
+          <Typography
+            font={{ family: 'ABCDiatype', weight: 'Regular' }}
+            className="text-xs uppercase"
+          >
+            posts
+          </Typography>
+
+          <View className="space-y-1">
+            <Typography
+              font={{ family: 'ABCDiatype', weight: 'Regular' }}
+              className="text-sm text-shadow text-right"
+            >
+              {formattedTotalPosts}
+            </Typography>
+          </View>
+        </View>
+      )}
+    </View>
+  );
+}
+
+function NetworkIcon({ chain }: { chain: Chain }) {
```

Comment:
we have a lot more chains now 😬 https://github.com/gallery-so/gallery/blob/main/packages/shared/src/utils/chains.ts

i just realized the icon path `/icons/X_logo.svg` works for web but not sure how that would work out for mobile. we might need some kind of ternary that handles dynamic importing based on the platform, like:

```js
  icon: isMobile ? '/icons/EthIcon' : '/icons/ethereum_logo.svg'
```

^ don't need to handle in this PR though

Reply:
Yup! Need to more chain support to mobile